Frontend/voitheia-App/src/app/components/layout/header/header.component.scss

```diff
@@ -1,49 +1,424 @@
 @import '../../../../variables.scss';
 
-:host{
-    nav{
-        height:$header-height;
-        width:100%;
-        background-color:$brand-background-color;
-        padding:$half-spacing;
-        position:absolute;
-        top:0;
-        z-index:105;
-        font-size:$large;
-    
-        i#logo{
-            height:100px;
-            width:100px;
-            background-image: url(../../../../assets/images/logo.svg);
-            background-repeat: no-repeat;
-            background-size: contain;
-            margin-right:$half-spacing;
-        }
-
-        h1{
-            font-size: 4em; 
-            font-weight: bold; 
-            display: inline-block;
-            cursor: pointer;
-            font-family: 'Kelson', sans-serif;
-        }
+*,
```

Comment:
Overall should we use the functionality of SCSS and nest different css tags. A best practice in Angular is to nest all the Css in a :host tag

Reply:
code was obsolete and thus removed